app/components/tag/component.tsx

```diff
@@ -3,17 +3,15 @@ import cx from 'classnames';
 
 export interface TagProps {
   children: React.ReactNode;
-  className?: string;
+  className?: string | unknown;
 }
 
-export const Tag: React.FC<TagProps> = ({
-  children,
-  className = 'text-black bg-gray-300',
-}: TagProps) => (
+export const Tag: React.FC<TagProps> = ({ children, className }: TagProps) => (
   <div
     className={cx({
       'relative inline-flex rounded': true,
-      [className]: !!className,
+      [`${className}`]: !!className,
```

Comment:
Though this works fine I was wondering if it would make to use a `prop` to set tag styles. Setting a `type` prop with some associated styles, like we are doing on the buttons, i.e:
`<Tag type={t.type}/>`  and having on the `interface` some `type: 'bioregional' | 'species' | 'source';`
I'm more inclined to do that because it would not allow devs to mess with the styles.
What do you think?

Reply:
As I don't know yet the different number of colors that we could have I'll leave it like that. But yes, you are right that, if we have a fixed number of them, we could implement them by theme